node/config/types.go

```diff
@@ -96,6 +96,10 @@ type ProviderSubsystemsConfig struct {
 	WindowPostMaxTasks  int
 	EnableWinningPost   bool
 	WinningPostMaxTasks int
+
+	EnableWebGui bool
+	// The address that should listen for Web GUI requests.
+	GuiAddress string
```

Comment:
Maybe a separate UI section? Initially the subsystems section was there for enabling/disabling high-level subsystems (tho I'm starting to think that we should rework the whole lotus-provider config, basing it on lotus config brings in some avoidable tech debt)

Reply:
Lets talk about the rework idea. It's a mess having "old, dont touch" subsections when deciding where to put things because the consistency is gone.